src/webui/api/synccontroller.h

```diff
@@ -28,6 +28,8 @@
 
 #pragma once
 
+#include <QElapsedTimer>
```

Comment:
When I try forward declaring the class (and moving the include to the .cpp) I get this error:

```
compiling moc_synccontroller.cpp
In file included from webui/webapplication.cpp:62:0:
webui/api/synccontroller.h:62:19: error: field ‘m_freeDiskSpaceElapsedTimer’ has incomplete type
     QElapsedTimer m_freeDiskSpaceElapsedTimer;
```

Reply:
You can't create variable (class field here) of incomplete class. Compiler need to know its definition to do it.